src/test/java/org/carlspring/cloud/storage/s3fs/util/BaseIntegrationTest.java

```diff
@@ -0,0 +1,25 @@
+package org.carlspring.cloud.storage.s3fs.util;
+
+import static org.carlspring.cloud.storage.s3fs.S3Factory.ACCESS_KEY;
+import static org.carlspring.cloud.storage.s3fs.S3Factory.SECRET_KEY;
+
+public abstract class BaseIntegrationTest
+{
+
+    static {
```

Comment:
Cosmetic: Braces on new lines please. :)

Reply:
Also, just wondering, if it wouldn't be better to do this via a JUnit extension as illustrated [here](https://github.com/carlspring/junit5-experiments/blob/master/junit5-before-all-extension/src/test/java/org/carlspring/junit/ExampleExtension.java)?

Reply:
Yeah, I thought about an extension but then I saw the `EnvironmentBuilder`. That could be included in that `BaseIntegrationTest` class I think.

Reply:
Also, it looks like you're assuming that all integration tests will always be using MinIO, which is not the case. Depending on what profile is enabled, it might be Amazon S3 and/or MinIO.

Is this just for the sake of the current experimentation, or...?

Reply:
Since I'm about to merge my PR and this one, I'm going to leave things "as-is". If we decide going the JUnit extension way - we can open up a follow-up task later. :)